at_commons/lib/src/verb/notify_verb_builder.dart

```diff
@@ -93,6 +97,9 @@ class NotifyVerbBuilder implements VerbBuilder {
       ccd ??= false;
       command += 'ttr:$ttr:ccd:$ccd:';
     }
+    if(isTextMessageEncrypted){
```

Comment:
please dart format this file

Reply:
Ran dart formatter and pushed the changes.